app/views/assignments/show.html.erb

```diff
@@ -32,206 +32,201 @@
+                            ),
+                            { class: 'button', method: :post,
+                              data: { confirm: confirm, disable_with: t(:please_wait) } } %>
+              </p>
+            <% end %>
             <p>
-              <%= link_to t('groups.student_interface.work_alone'),
-                          course_assignment_groups_path(
-                            course_id: @assignment.course_id,
-                            assignment_id: @assignment.id,
-                            workalone: 'true'
-                          ),
+              <%= link_to t('helpers.submit.create', model: Group.model_name.human),
+                          course_assignment_groups_path(@assignment.course_id, @assignment.id),
                           { class: 'button', method: :post,
-                            data: { confirm: confirm, disable_with: t(:please_wait) } } %>
+                            data: { confirm: confirm,
+                                    disable_with: t(:please_wait) } } %>
             </p>
           <% end %>
-          <p>
-            <%= link_to t('helpers.submit.create', model: Group.model_name.human),
-                        course_assignment_groups_path(@assignment.course_id, @assignment.id),
-                        { class: 'button', method: :post,
-                          data: { confirm: confirm,
-                                  disable_with: t(:please_wait) } } %>
-          </p>
-        <% end %>
-      <% else %>
-        <% # Since the student has a group already, we display information about the group only %>
-        <h3>
-          <%= t('groups.student_interface.your_group') %>
-        </h3>
-        <% # Display group properties are read only warning
-           # when the due date and grace period (if any) have passed.
-           if @assignment.past_collection_date?(@current_role.section) && @assignment.student_form_groups &&
-             !@assignment.invalid_override && @assignment.group_max > 1 %>
-          <p class='information'>
-            <%= t('groups.student_interface.modification_limited') %>
-          </p>
-        <% end %>
-
-        <% # If the group does not have the right amount of members yet, display
-           # 'group is not valid' warning.
-           unless @grouping.is_valid? %>
-          <p class='warning'>
-            <%= t('groups.student_interface.invite_more_students') %>
-          </p>
-        <% end %>
-
-        <%# For each member of this group, display name and status %>
-        <ul>
-          <% @grouping.student_memberships.includes(:role).each do |member|%>
-            <%= render partial: 'assignments/member',
-                       locals: { member: member,
-                                 grouping: @grouping } %>
-          <% end %>
-        </ul>
+        <% else %>
+          <% # Since the student has a group already, we display information about the group only %>
+          <% if @assignment.group_max > 1 %>
```

Comment:
As above, you no longer need this check. (Make sure to revert any relevant indentation changes as well.)